app/src/lib/stats/stats-store.ts

```diff
@@ -912,7 +921,7 @@ export class StatsStore implements IStatsStore {
   /**
    * Increments the `createPullRequestCount` metric
    */
-  public recordCreatePullRequest(): Promise<void> {
+  public recordCreatedPullRequest(): Promise<void> {
```

Comment:
This name change seems like it might just be cleanup for consistency? Just wanted to make sure it's not going to impact anything in the actual metrics before/after this change.

Reply:
Oops! That was totally a mistake. Thanks for catching it @billygriffin! Fixed in 8ad060c and bfba4ef

Reply:
Oops! That was totally a mistake. Thanks for catching it @billygriffin! Fixed in 8ad060c and bfba4ef